publication.gradle

```diff
@@ -29,3 +29,29 @@ kotlin {
         }
     }
 }
+
+apply plugin: 'com.jfrog.bintray'
+
+bintray {
+    user = findProperty('user')
+    key = findProperty('key')
+
+    pkg {
+        repo = 'maven'
+        name = 'reaktive'
+        userOrg = 'badoo'
+        vcsUrl = 'https://github.com/badoo/Reaktive.git'
+        licenses = ['Apache-2.0']
+        version.name = reaktive_version
+    }
+}
+
+// This is required for K/N publishing
+bintrayUpload.dependsOn publishToMavenLocal
```

Comment:
Can you use [Tasks Provider API](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html)? `tasks.getByName('bintrayUpload') { dependsOn tasks.getByName('publishToMavenLocal') }`. Same for next task.